src/Node/TransNode.php

```diff
@@ -96,6 +131,15 @@ public function compile(Compiler $compiler)
                     ->raw(', ');
             }
 
+            if ($hasContext) {
+                if (static::$hasContextFunctions || static::$enableMoTranslator) {
```

Comment:
```suggestion
            if ($hasContext && (static::$hasContextFunctions || static::$enableMoTranslator)) {
```

Reply:
I think that I will not apply this one, the code is complex enough to understand
Also I would say this is more readable like it is

Reply:
What about:
```suggestion
            if ($hasContext
                && (static::$hasContextFunctions || static::$enableMoTranslator)
            ) {
```

Reply:
I fixed it, sorry I was mistaken with the code at the end of the file that is similar
Applied your second suggestion that is better and readable